docs/store-operations/pricing/msf-international-enhancements.mdx

```diff
@@ -0,0 +1,93 @@
+# Price Lists - International Enhancements for Multi-Storefront
+
+<Callout type="info">
+  Tax-inclusive and exclusive price lists are in beta. Features may change based on user feedback or internal testing.
+</Callout>
+
+When you sell internationally, you may need your storefront to display prices that either include or exclude tax based on regional expectations. You may need some storefront channels to display prices that include tax, while others display prices that exclude tax.
+
+To localize your storefronts, you can do the following:
+- Specify whether a price list has prices that include or exclude tax when entered.
+- Assign the price list to storefront channels (or customer groups).
+
+
+## Limitations
+
+- Stores using [Catalog V2](/docs/store-operations/catalog/migration) have fixed pricing and will use global store settings. 
+- You cannot enter price changes due to product modifier rules, gift wrapping features, and fixed shipping prices into a price list. These inherit global settings and always apply on top of the price list.
+- You cannot mix inclusive and exclusive pricing within a single price list.
+- You must [set the display price](https://support.bigcommerce.com/s/article/Product-Display-Settings?language=en_US#settings) to the lowest variant price when a shopper's preselected options don't match to a price. 
+
+If your store doesn't have multiple storefronts, you can still create tax-inclusive or exclusive price lists and assign them to customer groups.
+
+## Settings
+
+Price lists will have a `prices_entered_with_tax` setting, which overrides the (default) global store setting. Use this setting to choose whether the price list is tax-inclusive or exclusive, or inherits the global settings. For details, see the [Price List endpoints](/docs/rest-management/price-lists) in the REST Management APIs.
+
+When you're using tax-inclusive price lists, tax-inclusive prices remain the same across all tax zones; the tax amount will vary based on the shopper's location. 
```

Comment:
This is only true if they turn on the setting I mentioned above- otherwise tax unwinding will kick in and sadness will ensue.

I _think_ this is being changed to make it the default for ex tax stores, but depending on when someone sees this doc that may not be completed/ totally rolled out.

Reply:
I added a sentence about turning on the setting. Before this doc is published, maybe we'll revisit this sentence to see whether it's rolled out

Reply:
I re-thought about the feature. My current understanding is:
- `When calculating the taxes on sales made outside the pricing tax zone` setting is only important when you want tax-inclusive prices (the setting doesn't matter if you want tax-exclusive prices). 
- When you want a tax-inclusive price list, there's no`When calculating the taxes on sales made outside the pricing tax zone` setting for the price list. So I'm guessing this pretty-pricing setting would inherit the global value (assuming the global store is also set to tax-inclusive).
- If you want a tax-inclusive price list and the global store is tax-exclusive, we'll have to confirm with @bigcommerce/team-tax what the default is. I'm actually taking a closer look at the Tax Settings API reference, and now I'm thinking the setting actually defaults to non-pretty pricing. Either way, my doc could clarify these points.

^ @benpratt77 can you comment on the default value?